go.mod

```diff
@@ -17,4 +17,5 @@ require (
 	k8s.io/utils v0.0.0-20200729134348-d5654de09c73
 	sigs.k8s.io/controller-runtime v0.6.3
 	sigs.k8s.io/controller-tools v0.4.0
+	sigs.k8s.io/service-apis v0.1.1-0.20210114050902-2b2bf1b8fdf5
```

Comment:
@stevesloka FYI I had to bump service apis to this version to get a fix for `gatewayclass.status`.

Reply:
Ahh ok cool! We'll have to bump Contour to match the same.